package.json

```diff
@@ -6,27 +6,41 @@
     "pino": "^8.11.0",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
+    "react-scripts": "5.0.1",
     "web-vitals": "^2.1.4"
   },
   "devDependencies": {
+    "@storybook/addon-essentials": "^7.0.0-beta.62",
+    "@storybook/addon-interactions": "^7.0.0-beta.62",
+    "@storybook/addon-links": "^7.0.0-beta.62",
+    "@storybook/blocks": "^7.0.0-beta.62",
+    "@storybook/preset-create-react-app": "^7.0.0-beta.62",
+    "@storybook/react": "^7.0.0-beta.62",
+    "@storybook/react-webpack5": "^7.0.0-beta.62",
+    "@storybook/testing-library": "^0.0.14-next.1",
     "@testing-library/jest-dom": "^5.16.5",
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
     "@types/jest": "^27.5.2",
     "@types/node": "^16.18.12",
     "@types/react": "^18.0.28",
     "@types/react-dom": "^18.0.11",
+    "babel-plugin-named-exports-order": "^0.0.2",
     "eslint-config-airbnb": "^19.0.4",
     "eslint-config-airbnb-typescript": "^17.0.0",
-    "react-scripts": "5.0.1",
-    "typescript": "^4.9.5"
+    "prop-types": "^15.8.1",
+    "storybook": "^7.0.0-beta.62",
+    "typescript": "^4.9.5",
+    "webpack": "^5.75.0"
```

Comment:
`webpack` is currently a transitive dependency of `react-scripts`; do we need to depend on it directly?

```suggestion
```